frontend/simple/views/Mailbox.vue

```diff
@@ -268,10 +249,8 @@ export default {
       mode: 'Inbox',
       recipient: null,
       recipients: [],
-      inbox: [],
-      invites: [],
       composedMessage: '',
-      currentMessage: new Events.PostMessage({from: null, message: ''}, null),
+      currentMessage: new Events.HashableMailboxPostMessage({from: null, message: ''}, null),
```

Comment:
What is this? A comment is necessary explaining why this is here and its purpose